utility/src/main/java/org/oppia/android/util/parser/html/ListItemLeadingMarginSpan.kt

```diff
@@ -153,30 +143,24 @@ sealed class ListItemLeadingMarginSpan : LeadingMarginSpan {
       val isFirstCharacter = startCharOfSpan == start
 
       if (isFirstCharacter) {
+        // Force left alignment
+        paint.textAlign = Paint.Align.LEFT
+
         val textWidth = Rect().also {
```

Comment:
Looks like this variable is nolonger used in the new code.

Reply:
Done.